README.md

```diff
@@ -1,2 +1,8 @@
 # Hello-Microverse
 Setting up linters
+
+Built With
+Html and css.
+
+Getting Started.
+In this project we are learning to follow the git flow and set up linter.
```

Comment:
- [ ] Please add a descriptive README file to your project - Kindly use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project ✔️

Reply:
I add a descriptive README to my project. I used the temple from Microverse and I improve my file.